### PR TITLE
monitoring: update node_pod_quota

### PR DESCRIPTION
**Pull request by huanggze**
Signed-off-by: huanggze <loganhuang@yunify.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
image

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/simple/client/monitoring/prometheus/testdata/promqls.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ var PromQLs = map[string]string{
 	"cluster_cpu_utilisation":               `:node_cpu_utilisation:avg1m`,
 	"node_cpu_utilisation":                  `node:node_cpu_utilisation:avg1m{node="i-2dazc1d6"}`,
 	"node_cpu_total":                        `node:node_num_cpu:sum{node=~"i-2dazc1d6|i-ezjb7gsk"}`,
+	"node_pod_quota":                        `max(kube_node_status_capacity{resource="pods",node=~"i-2dazc1d6|i-ezjb7gsk"}) by (node) unless on (node) (kube_node_status_condition{condition="Ready",status=~"unknown|false"} > 0)`,
```
Comment (zryfish) [inline pkg/simple/client/monitoring/prometheus/testdata/promqls.go:7]: what are those instance ids for

Comment (huanggze) [reply pkg/simple/client/monitoring/prometheus/testdata/promqls.go:7]: they are test code

**Pull request merged**